### ISSUE TITLE
Ugly format for private enum with @TestVisible annotation 

### ISSUE THREAD

**Issue by garychenming** [dangmai/prettier-plugin-apex#184]
**Input:**

```apex
@TestVisible
private enum AccountType {
    BUSINESS_ACCOUNT,
    PERSON_ACCOUNT
}
```

**Actual output:**

```apex
@TestVisible
 private enum AccountType {  // There is a useless space before private.
    BUSINESS_ACCOUNT,
    PERSON_ACCOUNT
}
```

**Expected output:**

```apex
@TestVisible
private enum AccountType {
    BUSINESS_ACCOUNT,
    PERSON_ACCOUNT
}
```

**Additional information (please fill this out)**:

- OS: OS X
- Version: [10.15.2]

**Comment by dangmai** [comment]
Thank you for the bug report! I'll try it out and see if I can reproduce it soon.

**Comment by dangmai** [comment]
I've fixed this issue and released new version `1.3.0`. 

Thanks again for the bug report!

**Comment by garychenming** [comment]
Thank you for your quick response.
